account_invoice_export/views/message_template.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="exception_sending_invoice">
+        <div> Error sending invoice to <t t-esc="transmit_method_name" />.
+            <div class="mt16">
+                <p>An error <b>
+                        <t t-esc="send_error" />
+                    </b> was returned by the distant server.</p>
+            </div>
+            <div class="mt16">
+                <p>The failed job has the uuid <t t-esc="job_id" /></p>
```

Comment:
You could put a `<a>` link that would open the job record, would be a nice addition. I don't remember the way but it has been done in core addons.

Reply:
Yes this is a good idea, also it may get a little tricky with the job id being an uuid, I need to investigate.
Also from some Odoo example (in `stock.picking`) the links generated in the chatter did not work as expected and would open the main Odoo page, not sure why.